test/results_test.rb

```diff
@@ -4,6 +4,8 @@
 module HeapProfiler
   class ResultsTest < Minitest::Test
     def test_full_results
+      # We need to define it because we are not in cli flow where it is set up
+      HeapProfiler::AbstractResults.top_entries_count = 50
```

Comment:
This can be solved by setting the default in `results.rb`:

```ruby
    @top_entries_count = 50
    class << self
      attr_accessor :top_entries_count
    end
```

Reply:
Yes, better idea!